web/src/main/java/org/cbioportal/web/StudyController.java

```diff
@@ -103,7 +103,7 @@ private void warmDefaultResponseCache() {
         // on portals using auth, as in auth setting, different users will have different
         // results.
         if (!usingAuth()
-                && appName.equals("public-portal")
+                && showAllCancerStudies
```

Comment:
Can't we remove this configuration? Since we are already checking whether we are using auth isn't this feature flag redundant? Seems like a good optimization for any portal that's running without authentication?